docker/Dockerfile.GitHubActions

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:bionic
+LABEL maintainer="Dash Developers <dev@dash.org>"
+LABEL description="Dockerised DashCore"
+
+ARG USER_ID
```

Comment:
Set default value

Reply:
Default values are set on lines 12 & 13

Reply:
Yeah, but I think it's more clear to set default values for arguments but not for envs interpolation. For example, in case if you use these args more than once in this Dockerfile

Reply:
I think this was done intentionally for [scoping reasons](https://docs.docker.com/engine/reference/builder/#using-arg-variables). Maybe it was done like that for some other CI like GitLab, Jenkins or Travis? Either way we don't use it in the GitHub Actions, so maybe it is better to just delete the `ARG`s?

Reply:
Let's keep just keep it as it is. I just saw in all examples that people use default values. It's not so important